.env-dev.template

```diff
@@ -0,0 +1,77 @@
+SERVER_HOST=${JB_API_SERVER_HOST}
+
+KAFKA_BROKER=${JB_KAFKA_BROKER}
```

Comment:
We have just KAFKA_BROKER and just POSTGRES_DATABASE_HOST

no prefixes JB_ and PWR_ required.